hydrus/parser/openapi-hydra_parser.py

```diff
@@ -0,0 +1,121 @@
+"""
+Module to take in Open Api Specification and convert it to HYDRA Api Doc
+
+"""
+import yaml
+import json
+
+from hydrus.hydraspec.doc_writer import HydraDoc, HydraClass, HydraClassProp, HydraClassOp
+
+with open("../samples/openapi_sample.yaml", 'r') as stream:
+    try:
+        doc = yaml.load(stream)
+    except yaml.YAMLError as exc:
+        print(exc)
+
+
+definitionSet = set()
```

Comment:
Declare these in the `getClasses()` function and pass it as a variable to `get_ops()`. Whenever you can replace global variables with passable parameters, you should opt for the latter. `hydra_doc` can be declared in `getClasses()` as well and can be returned from to main when function finishes execution.